### PR TITLE
add/delete word to/from List, see all list in a word, see word details

### PR DESCRIPTION
**Pull request by minhtien1403**
## [Handle Events for List]
- Press add in DetailsView to add current word to List
- Click to word in List to see word Details
### [Redmine link]
[34576](https://edu-redmine.sun-asterisk.vn/issues/34576)
#### Screenshots (nếu có)
Show List View as Pop up while click to add button in Details View
Screen Shot 2021-04-04 at 21 09 31

Click to List in List VIew to see all word in List
Screen Shot 2021-04-04 at 21 10 21

### CODE REVIEW

File: T-Dict/T-Dict/Controller/DetailsVC/DetailsViewController.swift [hunk 1]
```diff
@@ -106,6 +106,23 @@ class DetailsViewController: BaseViewController, UINavigationControllerDelegate
                                           completion: nil)
         currentPageIndex = nextIndex
     }
+    
+    @objc func didTapAddToList() {
+        guard let word = word else {
+            print("Failed to add")
+            return
+        }
+        guard let definceVC = childViews[0] as? DefinitionViewController else {
```
Comment (dongtg-1485) [inline T-Dict/T-Dict/Controller/DetailsVC/DetailsViewController.swift]: `childViews` mà empty thì func này sẽ gây crash

File: T-Dict/T-Dict/Controller/DetailsVC/DetailsViewController.swift [hunk 2]
```diff
@@ -106,6 +106,24 @@ class DetailsViewController: BaseViewController, UINavigationControllerDelegate
                                           completion: nil)
         currentPageIndex = nextIndex
     }
+    
+    @objc func didTapAddToList() {
+        guard let word = word else {
+            print("Failed to add")
+            return
+        }
+        guard !childViews.isEmpty, let definceVC = childViews[0] as? DefinitionViewController else {
```
Comment (datdt-1935) [inline T-Dict/T-Dict/Controller/DetailsVC/DetailsViewController.swift]:
```suggestion
        guard let definceVC = childViews.first as? DefinitionViewController else {
```

File: T-Dict/T-Dict/Controller/ListVC/ListItemVC/ListItemViewController.swift [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configTableView()
+    }
+    
+    func configTableView() {
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(cellType: ListItemCell.self)
```
Comment (datdt-1935) [inline T-Dict/T-Dict/Controller/ListVC/ListItemVC/ListItemViewController.swift]: dùng `do`

File: T-Dict/T-Dict/Controller/DetailsVC/DetailsViewController.swift (hunk 2 above)
Comment (datdt-1935) [inline T-Dict/T-Dict/Controller/DetailsVC/DetailsViewController.swift:115]:
```suggestion
        guard let defineVC = childViews.first as? DefinitionViewController else {
```

**Review by dongtg-1485 — APPROVED** [review]

**Review by datdt-1935 — APPROVED** [review]

**Pull request merged**